python_modules/dagster/dagster/core/log_manager.py

```diff
@@ -174,41 +257,64 @@ def logging_metadata(self) -> DagsterLoggingMetadata:
     def loggers(self) -> List[logging.Logger]:
         return self._loggers
 
-    def log_dagster_event(self, level: int, msg: str, dagster_event: "DagsterEvent"):
+    def begin_python_log_capture(self):
+        for logger in self._managed_loggers:
+            logger.addHandler(self._dagster_log_handler)
+
+    def end_python_log_capture(self):
+        for logger in self._managed_loggers:
+            logger.removeHandler(self._dagster_log_handler)
+
+    def log_dagster_event(self, level: Union[str, int], msg: str, dagster_event: "DagsterEvent"):
```

Comment:
I don't see this called anywhere. Who is expected to call it?

Reply:
This functionality was added in a different PR (it's just been moved/given a better docstring in this PR). It's called in a few places (for example: https://github.com/dagster-io/dagster/blob/4ef3b5d1784e5ea818f8088ad1804ff33eb406c0/python_modules/dagster/dagster/core/events/__init__.py#L207).